package.json

```
@@ -1,8 +1,8 @@
 {
   "name": "@openmrs/esm-root-config",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

I recommend not incrementing the `version` number in PRs like this because the best/easiest way is to let `npm version minor` do it for you right before publishing it. Advantages to doing it through `npm version minor`:

1. `npm version minor` creates an annotated git tag, which conveniently turns into a github release when pushed, that can be used to send email/desktop notifications to all watchers of releases.
2. `npm version minor` creates a git commit that increments the version number with a standard commit message that you can find in thousands of npm packages. It makes looking at the commit history easy because you know exactly when 1.2.0 was because it has an entire commit dedicated to it.
3. It's best to leave it up to the maintainer/publisher to determine whether these changes warrant a major, minor, or patch version bump.

```suggestion
  "version": "1.1.1",
```


```